src/main/java/seedu/duke/parsermanager/ParseCheckProperty.java

```diff
@@ -43,7 +43,7 @@ public Command parseCommand() throws ParseCheckException {
     private ArrayList<String> processCommandDetails(String rawCommandDetail) throws CheckPropertyMissingFlagException,
             CheckExtraParametersException {
 
-        String[] flags = CHECK_PROPERTY_FLAGS;
+        String[] flags = INDEX_FLAGS;
```

Comment:
Is index flags multiple flags or only one flag?
Because if there is only 1 flag in it, you can consider using a string instead of an array